links.html

```diff
@@ -5,6 +5,11 @@
     <title>YangZhou</title>
     <meta charset="utf-8">
     <link rel="stylesheet" href="styles/main.css" type="text/css">
+    <style>
+        #useful-links>ul>li>a {
+            color: blueviolet;
+        }
+    </style>
```

Comment:
You put this in your external stylesheet, but the instructions asked for an embedded stylesheet as you see here